src/rulesets/oas/index.json

```diff
@@ -418,10 +419,9 @@
       "type": "validation",
       "given": "$..parameters..[?(@.in == 'body')]..[?(@property !== 'properties' && @.example && ( @.type || @.format || @.$ref ))]",
       "then": {
-        "function": "schemaPath",
+        "function": "oasExample",
```

Comment:
@P0lip My understanding is that oas2 only supports `examples` on responses. Woudn't this try and validate an `examples` field here would it exist?

Reply:
Yeah, it'd, but we can ignore it out if needed.

Reply:
@P0lip maybe passing, per rule, a whitelist of valid fields as functionOptions?

Reply:
Hmmm, so if I understand it correctly, we'd only need to support `responses` in case of OAS2.
@philsturgeon correct?

Reply:
@P0lip sorry friend I don't understand the question.

Reply:
so what I meant was whether OAS2 allows you to place `examples` under other spots than just responses.

Reply:
I think this thread makes it about as clear as I could hope to make it, but it’s a very confusing situation.

Reply:
https://github.com/stoplightio/studio/issues/381#issuecomment-626944301

Reply:
Yeah, I went through it, but still feels like a nightmare to maintain :sweat_smile: 
In any case, I'll try to come up with something decent enough that should cover most of cases we have to support.